symphony/lib/toolkit/class.entrymanager.php

```diff
@@ -116,7 +116,13 @@ public static function add(Entry $entry)
                 continue;
             }
 
-            Symphony::Database()->delete('tbl_entries_data_' . $field_id, sprintf("
+            // Check if table exists
+            $table_name = 'tbl_entries_data_' . General::intval($field_id);
+            if (!$field->requiresTable() || !Symphony::Database()->tableExists($table_name)) {
```

Comment:
@nitriques since you put or as a condition; wouldn't `requiresTable()` be completely useless if it returns false as it will always or with the table exists? Or is the idea to be 100% sure that there is no table to delete?

I'm fine with the PR just a question.